subs.py

```diff
@@ -23,21 +23,26 @@
 
 
 def get_subtitles(video_id):
-    mediaelement_json = requests.get("http://psapi-granitt-prod-we.cloudapp.net/mediaelement/%s" % (video_id)).json()
-    if not mediaelement_json["hasSubtitles"]:
+    mediaelement_json = requests.get("https://psapi.nrk.no/playback/manifest/%s" % (video_id)).json()
+
+    if not mediaelement_json["playable"]['subtitles']:
         return None
-    ttml_sub_url = mediaelement_json["subtitlesUrlPath"]
-    html = requests.get(ttml_sub_url).text
-    if not html:
+    ttml_sub_url = mediaelement_json["playable"]['subtitles'][0]['webVtt']
+    subs = requests.get(ttml_sub_url).text
+    if not subs:
         return None
 
-    content = _ttml_to_srt(html)
+    content = vtt_to_srt(subs)
```

Comment:
```suggestion
    content = _vtt_to_srt(subs)
```
please make the the function private